server/routes_test.go

```diff
@@ -673,3 +652,38 @@ func TestNormalize(t *testing.T) {
 		})
 	}
 }
+
+func TestVersionHandler(t *testing.T) {
```

Comment:
after some cleanup below, this could be a table-test that reads compactly like `tt := []struct{ method string, want string }...`